src/ontology/mondo-edit.obo

```diff
@@ -108056,9 +108060,6 @@ def: "Blocking of a blood vessel by cholesterol-rich atheromatous deposits, gene
 subset: otar {source="MONDO:OTAR"}
 synonym: "atheroembolism" EXACT [DOID:1461]
 synonym: "cholesterol crystal embolism" EXACT [DOID:1461]
-synonym: "purple toe syndrome" EXACT [DOID:1461]
-synonym: "trash foot" EXACT [DOID:1461]
-synonym: "warfarin blue toe syndrome" EXACT [DOID:1461]
```

Comment:
I agree that these should not be exact, even though they are reported as synonym in DO. 
Instead of removing them, we should update the synonym scope to "narrow synonym" since the synonyms refers to a child term of this term.